tools/clang/test/DXC/dxa_tests.hlsl

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Probably worth giving this a `.test` extension instead of `.hlsl` and ensuring that `.test` is supported in the suite. This isn't actually a source file.